arch/arm/boot/dts/overlays/revpi-connect4-overlay.dts

```diff
@@ -22,10 +22,10 @@
 				connect-gpios = <&gpio 6 GPIO_ACTIVE_HIGH>,
 						<&gpio 16 GPIO_ACTIVE_HIGH>;
 
-				left-sniff-gpios = <&expander_core 8 GPIO_ACTIVE_HIGH>,
-						   <&expander_core 9 GPIO_ACTIVE_HIGH>;
-				right-sniff-gpios = <&expander_core 11 GPIO_ACTIVE_HIGH>,
-						   <&expander_core 12 GPIO_ACTIVE_HIGH>;
+				left-sniff-gpios = <&expander_core 9 GPIO_ACTIVE_HIGH>,
+						   <&expander_core 8 GPIO_ACTIVE_HIGH>;
+				right-sniff-gpios = <&expander_core 12 GPIO_ACTIVE_HIGH>,
+						   <&expander_core 11 GPIO_ACTIVE_HIGH>;
```

Comment:
Could you please add some context? Were thy wrongly done in the devicetree or was there an error in the schematics?

Reply:
In the schematics the pins are named "L_SNIFF" and "L_ACK" while in the piControl Source Code we use the names "1A" and "2A" and both pins are called "Sniff-Pins". So its a bit hard to determine which one is which. However in code 1A is used to signal the start and end of the configuration while 2A is used to signal the presence of the Master (core or connect) and then to wait for a signal from a device. So IMHO 1A could be referred to as L_ACK and 2A as L_SNIFF. And in this case the schematics are wrong and the device tree was wrong before this change.